docs/agent-vs-aea.md

```diff
@@ -77,58 +77,58 @@ class MyAgent(Agent):
 
 ## Instantiate an `Agent`
 
-``` python
-# Ensure the input and output files do not exist initially
-if os.path.isfile(INPUT_FILE):
-    os.remove(INPUT_FILE)
-if os.path.isfile(OUTPUT_FILE):
-    os.remove(OUTPUT_FILE)
-
-# Create an addresses identity:
-identity = Identity(name="my_agent", address="some_address")
-
-# Set up the stub connection
-stub_connection = StubConnection(
-    input_file_path=INPUT_FILE, output_file_path=OUTPUT_FILE
-)
-
-# Create our Agent
-my_agent = MyAgent(identity, [stub_connection])
+```python
+    # Ensure the input and output files do not exist initially
```

Comment:
why tabs here?

Reply:
We need to have the blocks identical with the script to be able to compare them.